src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/VxlanFlowSpec.groovy

```diff
@@ -380,24 +390,30 @@ class VxlanFlowSpec extends HealthCheckSpecification {
         }
 
         when: "Try to create a VXLAN flow"
-        def flow = flowHelperV2.randomFlow(switchPair)
-        flow.encapsulationType = VXLAN.toString()
-        flowHelperV2.addFlow(flow)
+        def flowEntity = flowFactory.getBuilder(switchPair)
+                .withEncapsulationType(FlowEncapsulationType.VXLAN)
+                .build()
+        flowEntity.create()
 
         then: "Human readable error is returned"
         def createError = thrown(HttpClientErrorException)
         createError.rawStatusCode == 400
         def createErrorDetails = createError.responseBodyAsString.to(MessageError)
         createErrorDetails.errorMessage == "Could not create flow"
-        createErrorDetails.errorDescription == getUnsupportedVxlanErrorDescription("source", switchPair.src.dpId,
-                [FlowEncapsulationType.TRANSIT_VLAN])
+        createErrorDetails.errorDescription == getUnsupportedVxlanErrorDescription(
+                "source", switchPair.src.dpId, [FlowEncapsulationType.TRANSIT_VLAN])
 
```

Comment:
@yuliiamir This version does not have the `FlowNotCreateExpectedError` comparison approach yet. Let's have a call tomorrow, I have some questions about implementation.

Reply:
Done